ddtrace/internal/datadog/profiling/stack_v2/__init__.py

```diff
@@ -0,0 +1,33 @@
+try:
+    from ._stack_v2 import *  # noqa: F401, F403
+
+    def is_available():
+        # type: () -> bool
+        return True
+
+except Exception as e:
+    from ddtrace.internal.logger import get_logger
+
+    LOG = get_logger(__name__)
+    LOG.debug("Failed to import _stack_v2: %s", e)
+
+    def is_available():
+        # type: () -> bool
+        return False
+
+    # Decorator for not-implemented
+    def not_implemented(func):
```

Comment:
Perhaps we can define this once in `ddtrace.internal.compat`?

Reply:
Good call, although fixing this up would be a slight detour (I made testing annoying by fencing this failure path off).  I think we might consolidate these interfaces differently in the near-term anyway.